rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -159,12 +159,11 @@ def gather_candidates(self, context):
         try:
             script = self.get_script(source, line, col, filename,
                                      environment=self._env)
+            completions = self.get_completions(script)
         except ValueError:
             # Ignore ValueError
             return []
 
-        completions = self.get_completions(script)
-
```

Comment:
Same question as before.

Reply:
ValueError(invalid column) is generated from jedi.
It seems from asynchronous.

Reply:
The fix is wellcome.  I think the better check is needed.